examples/dts/requirements.txt

```diff
@@ -0,0 +1,8 @@
+autopep8
+grpcio>=1.60.0 # 1.60.0 is the version introducing protobuf 1.25.X support, newer versions are backwards compatible
+protobuf
+pytest
```

Comment:
`pytest` isn't a requirement for running the xample itself, and neither is `pytest-cov`

If you need to install this - move it to ` dev-requirements.txt` elsewhere?